NitroxLauncher/Patching/QModHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.IO;
+using NitroxModel.Logger;
+
+namespace NitroxLauncher.Patching
+{
+    public static class QModHelper
+    {
+        private static readonly string originalFileName = "winhttp.dll";
```

Comment:
minor: const

Reply:
Const cant be used here as static and readonly cant be used in conjunction with const

Reply:
A const is by default a static so you can remove both `static` and `readonly` to make this work.

Reply:
Good to know, will make the change